insights-notification-schemas-java/src/main/resources/schemas/Action.json

```diff
@@ -0,0 +1,123 @@
+{
+  "type": "object",
+  "title": "Notification message",
+  "comment": "Generated with https://hellosean1025.github.io/json-schema-visual-editor/",
```

Comment:
Thanks Sean! 😄

Reply:
I actually had to add the comment myself to remember the tool I used :D